src/execution/vm/vm.cpp

```diff
@@ -577,14 +577,14 @@ void VM::Interpret(const uint8_t *ip, Frame *frame) {
   }
 
   OP(ParallelScanTable) : {
-    auto db_oid = READ_UIMM4();
+    // TODO(Yuhong): might need to change function arguments
     auto table_oid = READ_UIMM4();
-    auto query_state = frame->LocalAt<void *>(READ_LOCAL_ID());
-    auto thread_state_container = frame->LocalAt<sql::ThreadStateContainer *>(READ_LOCAL_ID());
+    //    auto query_state = frame->LocalAt<void *>(READ_LOCAL_ID());
```

Comment:
It looks like these can now be removed safely based on the arguments in place

Reply:
Updated